src/to_substrait.cpp

```diff
@@ -1313,6 +1313,22 @@ substrait::Rel *DuckDBToSubstrait::TransformGet(LogicalOperator &dop) {
 	return get_rel;
 }
 
+static substrait::Expression_Literal GetLiteralFromSubstraitExpression(const substrait::Expression &expr) {
```

Comment:
can you make this a class member?

substrait::Expression_Literal DuckDBToSubstrait::ToExpressionLiteral